Dockerfile

```diff
@@ -54,7 +54,15 @@ ENV \
     PHP_DISPLAY_ERRORS="1" \
     PHP_MEMORY_LIMIT="-1" \
     XHGUI_MONGO_URI="global-xhgui:27017" \
-    XHGUI_PROFILING="enabled"
+    XHGUI_PROFILING="enabled" \
+    TZ=Europe/Berlin \
+    PHP_DATE_TIMEZONE=${TZ} \
+    SET_CONTAINER_TIMEZONE=true \
```

Comment:
needed?

Reply:
yes, sadly

Reply:
where?

Reply:
https://www.ivankrizsan.se/2015/10/31/time-in-docker-containers/
using those two env's in production works flawless (without having to change any Dockerfile)

Reply:
only if you add this to your Container:
![image](https://user-images.githubusercontent.com/471387/66850670-8e5c3680-ef79-11e9-98a5-723caf04e1bc.png)
What we do not do

Reply:
remove

Reply:
```suggestion
```